go.mod

```diff
@@ -26,3 +26,5 @@ require (
 	gopkg.in/yaml.v2 v2.2.2
 	vitess.io/vitess v3.0.0-rc.3.0.20190602171040-12bfde34629c+incompatible
 )
+
+go 1.13
```

Comment:
My understanding is that this allows the code to use go 1.13 features, but we are supposed to support the last 2 stable versions. Should we set this to 1.12 instead?

Reply:
Oops, my bad. I use go 1.13 by default and it changes the `go.mod` automatically, so I guess I did a `git add -a` and included it. I'll remove it. Thanks for the catch!